lib/tasks/foreman_templates_tasks.rake

```diff
@@ -24,6 +27,23 @@ namespace :templates do
     puts results.join("\n")
   end
 
+  task :import => :sync
+
+  task :export => :environment do
+    ForemanTemplates::TemplateExporter.new({
+      verbose:   ENV['verbose'],
+      repo: ENV['repo'],
+      branch: ENV['branch'],
+      prefix: ENV['prefix'],
+      dirname: ENV['dirname'],
+      filter: ENV['filter'],
+      # associate: ENV['associate'],
+      metadata_export_mode: ENV['metadata_export_mode'],
+    }).export!
+
+    puts 'Export finished'
```

Comment:
I see the internal code logs using `logger` - but just this single line for rake. Can we get output parity here (at least using `@verbose`)?